scss/_patterns_search-and-filter.scss

```diff
@@ -19,7 +17,6 @@
       flex-wrap: wrap;
       height: auto;
       margin: 0;
-      min-height: $height;
       overflow: hidden;
       padding-left: $sph-inner--small;
       padding-right: $sp-x-large; // reserve space for selected counter
```

Comment:
ideally, this should be expressed as the sum of the icon positioned there + the margins on either side.